docs/concept/buckconfig.soy

```diff
@@ -2379,7 +2379,7 @@ cxx_library(
 {call buckconfig.entry}
   {param section: 'kotlin' /}
   {param name: 'kotlin_home' /}
-  {param example_value: '/usr/local/Cellar/kotlin/1.1.1' /}
+  {param example_value: '/usr/local/Cellar/kotlin/1.2.21' /}
```

Comment:
This change doesn't bring any value, does it?

Reply:
It actually does, instigating people to use most recent versions is very good. `1.1.1` is very outdated already :)

Reply:
For the purpose of documentation it doesn't matter